.jshintignore

```diff
@@ -1,5 +1,7 @@
 **/vendor
-node_modules
 cms/static/js/i18n/**/*.js
 lms/static/js/i18n/**/*.js
+lms/static/lms/js/build.js
+lms/static/lms/js/spec/main.js
```

Comment:
Why ignore `lms/static/lms/js/spec/main.js` and `lms/static/lms/js/build.js`? (I'm still unclear on the boundary between what jslint and jshint each do - are these files being linted but not hinted?)